sdk/textanalytics/Azure.AI.TextAnalytics/src/SentenceSentiment_internal.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    [CodeGenModel("SentenceSentiment")]
+    internal partial struct SentenceSentiment_internal
+    {
+        /// <summary> The sentence text. </summary>
+        public string Text { get; }
+        /// <summary> The predicted Sentiment for the sentence. </summary>
+        public string Sentiment { get; }
```

Comment:
Changed type from enum `SentenceSentimentValue` to `string`.
Reason => avoid breaking change so doing custom parsing